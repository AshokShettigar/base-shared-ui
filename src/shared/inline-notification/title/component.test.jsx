import { shallow } from 'enzyme';
import InlineNotificationTitle from './component';


const defaultProps = {
    children: 'Inline Notification Title'
};

describe('Inline Notification Title', () => {
    it('default renders', () => {
        const wrapper = shallow(<InlineNotificationTitle {...defaultProps} />);
        expect(wrapper).toMatchSnapshot();
    });

    it('renders with prop className', () => {
        const customProps = {
            className: 'class-name'
        };

        const wrapper = shallow(<InlineNotificationTitle {...defaultProps} {...customProps} />);

        expect(wrapper).toMatchSnapshot();
    });
});
