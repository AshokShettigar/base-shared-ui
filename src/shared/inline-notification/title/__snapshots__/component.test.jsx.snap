// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inline Notification Title default renders 1`] = `
<div
  className="inline-notification__title"
  data-component="InlineNotificationTitle"
>
  Inline Notification Title
</div>
`;

exports[`Inline Notification Title renders with prop className 1`] = `
<div
  className="inline-notification__title class-name"
  data-component="InlineNotificationTitle"
>
  Inline Notification Title
</div>
`;
